pandas/_libs/tslibs/period.pyx

```diff
@@ -1645,6 +1646,13 @@ cdef class _Period(object):
             elif other is NaT:
                 return NaT
             elif util.is_integer_object(other):
+                warnings.warn("Addition of integers to {cls} is "
```

Comment:
can you call a function for this warning (not sure exatcly where), rather than repeating each time, prob need to pass in the level

Reply:
Not sure what you have in mind.  Just a one-liner containing this warning?

Reply:
yeah a module level function just seems better than copy-pasting

Reply:
can you do this